<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VtkVisTabWidgetBase</class>
 <widget class="QWidget" name="VtkVisTabWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>259</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="margin">
    <number>3</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="VtkVisPipelineView" name="vtkVisPipelineView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoExpandDelay">
       <number>1</number>
      </property>
      <property name="indentation">
       <number>5</number>
      </property>
      <property name="animated">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="actorPropertiesGroupBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>180</height>
          </size>
         </property>
         <property name="title">
          <string>Actor Properties</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="margin">
           <number>3</number>
          </property>
          <item row="0" column="1">
           <widget class="ColorPickerPushButton" name="diffuseColorPickerButton">
            <property name="text">
             <string>(255,255,255)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Diffuse Color</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="visibleEdgesCheckBox">
            <property name="text">
             <string>Visible Edges</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="ColorPickerPushButton" name="edgeColorPickerButton">
            <property name="text">
             <string>(255, 255, 255)</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Opacity</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QValueTooltipSlider" name="opacitySlider">
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="activeScalarComboBox"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Active Scalar</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="transformTabWidget">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="scalingTab">
          <attribute name="title">
           <string>Scaling</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="spacing">
            <number>5</number>
           </property>
           <property name="margin">
            <number>5</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>5</number>
             </property>
             <property name="sizeConstraint">
              <enum>QLayout::SetDefaultConstraint</enum>
             </property>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Scaling Factor</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="scaleZ">
               <property name="text">
                <string>1</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="translationTab">
          <attribute name="title">
           <string>Translation</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <property name="spacing">
            <number>5</number>
           </property>
           <property name="margin">
            <number>5</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>5</number>
             </property>
             <item>
              <widget class="QLabel" name="transXLabel">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>X</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="transX">
               <property name="text">
                <string>0</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="transYLabel">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Y</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="transY">
               <property name="text">
                <string>0</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="transZLabel">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Z</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="transZ">
               <property name="text">
                <string>0</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="filterPropertiesGroupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="title">
          <string>Filter Properties</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QScrollArea" name="filterPropertiesScrollArea">
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="widgetResizable">
             <bool>false</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>263</width>
               <height>219</height>
              </rect>
             </property>
             <layout class="QFormLayout" name="formLayout">
              <property name="fieldGrowthPolicy">
               <enum>QFormLayout::FieldsStayAtSizeHint</enum>
              </property>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorPickerPushButton</class>
   <extends>QPushButton</extends>
   <header>ColorPickerPushButton.h</header>
  </customwidget>
  <customwidget>
   <class>VtkVisPipelineView</class>
   <extends>QTreeView</extends>
   <header>VtkVisPipelineView.h</header>
  </customwidget>
  <customwidget>
   <class>QValueTooltipSlider</class>
   <extends>QSlider</extends>
   <header>QValueTooltipSlider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
